app/serializers/invite_link_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class InviteLinkSerializer < ApplicationSerializer
+  attributes :id, :invite_link, :created_at, :max_redemptions_allowed, :redemption_count, :expires_at, :group_names
+
+  def invite_link
+    "#{Discourse.base_url}/invites/#{object.invite_key}"
```

Comment:
Is `Discourse.base_url` available on the client side? If so, we can avoid sending down the same string multiple times and have the client form it.